pypeit/scripts/coadd_datacube.py

```diff
@@ -319,7 +339,8 @@ def coadd_cube(files, parset, overwrite=False):
     primary_hdu = fits.PrimaryHDU(header=spec2DObj.head0)
```

Comment:
Consider making this a simple DataContainer.
You could model it after `OneSpec` in `coadd1d.py`